meter/dsmr.go

```diff
@@ -151,11 +156,14 @@ func (m *Dsmr) run(conn *bufio.Reader, done chan struct{}) {
 			conn, err = m.connect()
 			if err != nil {
 				handle("connect", err)
-				time.Sleep(time.Second)
+				sleep := backoff.NextBackOff()
+				log.INFO.Printf("Next attempt after: %s", sleep)
```

Comment:
if you really need to log this then in TRACE or DEBUG, but not INFO. Log messages always lower case and %v instead of %s. Maybe combine with logging the error above?

Reply:
fixed